src/main/java/careconnect/logic/commands/DeleteLogCommand.java

```diff
@@ -92,9 +94,9 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         if (this.requireConfirmation) {
-            // Add a DeleteLogCommand to the stack
-            Command.STACK.add(new DeleteLogCommand(personIndex, logIndex, false));
-            return new CommandResult(Command.CONFIRMATION_MESSAGE,
+            // Queues a DeleteLogCommand
+            Command.commandToConfirm = new DeleteLogCommand(personIndex, logIndex, false);
```

Comment:
This looks repeated in every Command, is it possible to extract this out into another function? Maybe a function in the parent Command class that takes in a Command (in this case DeleteLogCommand)?

Reply:
Prolly not important tho hahaha